### PR TITLE
Setup db and models

### PR DESCRIPTION
**Pull request by deborahleehamel**
adds migrations for all tables needed with corresponding relationship references
customers
merchants
invoices
transactions
items
invoice_items

### CODE REVIEW

File: spec/factories/items.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :item do
```
Comment (karinamzalez) [inline spec/factories/items.rb]: we should delete all of these factories if we're using fixtures

File: db/schema_view.xml [hunk 2]
```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (karinamzalez) [inline db/schema_view.xml:1]: Thanks for adding this!

File: spec/factories/items.rb (hunk 1 above)
Comment (deborahleehamel) [reply spec/factories/items.rb]: done. removed old factories files. Thanks!

File: app/models/merchant.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class Merchant < ApplicationRecord
+  has_many :invoices
+  has_many :items
+  has_many :invoice_items, through: :invoices
```
Comment (karinamzalez) [inline app/models/merchant.rb:4]: great attention to detail! (indentation)

**Pull request merged**